ui/src/pages/Input/InputPage.tsx

```diff
@@ -268,6 +323,7 @@ function InputPage() {
                             page={PAGE_INPUT}
                             handleOpenPageStyleDialog={handleOpenPageStyleDialog}
                             displayActionBtnAllRows
+                            serviceName=""
```

Comment:
seems like a type workaround. Should serviceName be optional instead of empty string?

Reply:
resolved